ote/resources/public/language/en.edn

```diff
@@ -275,7 +275,8 @@
 
   :RAE-link-text "RAE-software"
   :SEA-ROUTE-link-text "Sea traffic route and timetable editor"
-  :companies-main-info "If there are other companies involved in providing the service or the service if a brokering service, fill in all the names and business IDs of the companies involved. Only the companies coordinating the service fill in this part (e.g. taxi centres or transport producing authorities). Small companies who only take ride orders from taxi centres do not fill this part."
+  :companies-main-info (:md "If there are other companies involved in providing the service or the service if a brokering service, fill in all the names and business IDs of the companies involved. Only the companies coordinating the service fill in this part (e.g. taxi centres or transport producing authorities). Small companies who only take ride orders from taxi centres do not fill this part.
```

Comment:
"If there are other companies involved in providing the service or the service if a brokering service"
-> "If there are other companies involved in providing the service or the service IS a brokering service"